recipes/bornagain/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "BornAgain" %}
+{% set version = "1.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://apps.jcns.fz-juelich.de/src/BornAgain/BornAgain-{{ version }}.tar.gz
+  sha256: 20e390b683b3036aabf48d266e5e74da46ce9d781e72379dccb3d227ed300d40
+
+build:
+  number: 0
```

Please skip Python 2 builds explicitly.

```suggestion
  number: 0
  skip: true  # [py2k]
```